org.gnome.Cowsay.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "org.gnome.Cowsay",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.32",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "cowsay",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland"
+    ],
+    "build-options" : {
```

Comment:
This can be removed, 3.32 ships its own build-options (inherited from freedesktop sdk).